robot_comedy_ml/pitch_inten_avg_std_minmax/to_comedy_data.py

```diff
@@ -34,23 +34,30 @@
 
 # use performanceLog json file to list joke names in order they were told in the performance
 joke_order = []
-file = open("1.txt", "r", encoding='utf_16_le')
-path = file.read().splitlines()[0][:-16] # get path to the performance directory
-file.close()
-json_name = [i for i in os.listdir(path) if os.path.isfile(os.path.join(path,i)) and 'performanceLog' in i] # find name of json file
-json_path = path+json_name[0]
-json_file = open(json_path, "r")
-json_data = json.loads(json_file.read().splitlines()[0]) # read performanceLog json
-json_file.close()
-for entry in json_data:
-    keys = entry.keys()
-    if ("audio" in keys) and (entry.get("audio") != "/../sounds/robot_name_joke_1.ogg.wav"):
-	    joke_name = entry.get("audio")[11:] # find all joke names in the json and save them in order
-	    joke_order.append(joke_name)
-#print(joke_order) # uncomment to verify the resulting list
+numfiles = len([f for f in os.scandir(os.getcwd()) if f.name.endswith('.txt')]) # check how many txt files the praat script created
+for i in range(1, numfiles + 1):
+    joke_performance = []
+    file = open(str(i) + ".txt", "r", encoding='utf_16_le')
+    path = file.read().splitlines()[0][:-16] # get path to the performance directory
+    #print(path)
```

Comment:
Just for cleanliness, I would remove this commented line or leave a note like you did on line 56